src/services/structures/controllers/getDossiersReconventionnement.ts

```diff
@@ -0,0 +1,223 @@
+import { Application } from '@feathersjs/express';
+import { Response } from 'express';
+import { GraphQLClient } from 'graphql-request';
+import { IDossierDS, IRequest } from '../../../ts/interfaces/global.interfaces';
+import validReconventionnement from '../../../schemas/reconventionnement.schemas';
+import {
+  queryGetDemarcheReconventionnement,
+  queryGetDemarcheReconventionnementWithoutAttributDossier,
+} from '../repository/reconventionnement.repository';
+import TypeDossierReconventionnement from '../../../ts/enum';
+import { getNameStructure } from '../repository/structures.repository';
+
+const categoriesCorrespondances = require('../../../../datas/categorieFormCorrespondances.json');
+
+const getDemarcheNumber = (type: string) =>
+  categoriesCorrespondances.find((categorie) => categorie.type === type)
+    .demarche_number;
+
+const requestGraphQLForGetDemarcheDS = (
+  graphQLClient: GraphQLClient,
+  limitDossier: number,
+  paginationCursor: String,
+  type: string,
+) =>
+  graphQLClient
+    .request(queryGetDemarcheReconventionnement(limitDossier), {
+      demarcheNumber: getDemarcheNumber(type),
+      after: paginationCursor,
+    })
+    .catch(() => {
+      return new Error("La démarche n'existe pas");
```

Comment:
ok du coup on sort pas ici en res send not found ?

Reply:
non